src/covid_shared/cli_tools.py

```diff
@@ -29,6 +29,20 @@
 }
 
 
+def _remove_secrets(dictionary: Dict, secrets: List[str]) -> Dict:
+    """
+    Remove secrets from a nested Dict
```

Comment:
NIT: Naming.   You're not removing secrets, but censoring them.  Call it `censor_secrets` or `mask_secrets`.